Artifacts.toml

```diff
@@ -1,12 +1,21 @@
+# when generating a new artifact, add the new feather file to the datasets folder at
+# https://osf.io/a94tr/, then download that folder to generate the tarball
+# (so that we don't forget datasets)
+
 [TestData]
-git-tree-sha1 = "6614babc98083a65c25b7ec1ef2e38b345de65c9"
+# this is black magic to compute. Just let it error and copy over the new value.
+git-tree-sha1 = "4d1410cd290622e426411273ef379fe82b749ca4"
```

Comment:
```julia
 ┌ Error: Tree Hash Mismatch!
│   Expected git-tree-sha1:   e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
│   Calculated git-tree-sha1: 4d1410cd290622e426411273ef379fe82b749ca4
```

The documentation for artifacts needs to be improved a LOT.

Reply:
@Nosferican Do you recall if this is related to a version number issue?

Reply:
It's a bug in Pkg which computes different git-tree-sha1 depending on the platform if not mistaken. Last I heard, the Pkg team was planning on getting away from it or something but the current patch was to ignore that error. It is great for consistency and validating the content but it is currently broken.

Reply:
@Nosferican That's an additional bug -- look at the calculated SHA and the SHA in the diff. For some reason, Pkg is expected a different one than the one we have explicitly in the file  and which it is actually computing. :frowning:

Reply:
Checking if that issue is still present. Give it a shot and copy the one it is expecting? Might help us identify if it is a version mismatch.

Reply:
This is what I was thinking of https://github.com/JuliaLang/Pkg.jl/pull/1885.